app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt

```diff
@@ -453,3 +458,5 @@ enum class ScreenMode {
     WAKE_UP, // wake up the device on navigating to that NavigationItem (i.e IncomingCall)
     NONE // do not wake up and allow device to sleep
 }
+
+private const val MESSAGE_DETAILS_REACTIONS_LEARN_MORE_URL = "https://support.wire.com/hc/en-us/articles/212053645-Like-a-message"
```

Comment:
I'm not really sure where I could have added this URL to, if you have any suggestions please let me know so we can rearrange it :)

Reply:
We are adding this on `strings.xml` file, under "non-translatable" resources, you can find at leaset 2 more URL's there :D